packages/@okta/vuepress-site/docs/guides/pwd-optional-sign-in-email/main/nodeexpress/bestpractices.md

```diff
@@ -0,0 +1 @@
+[Recommended best practices](/docs/guides/pwd-optional-best-practices/nodeexpress/main/)
```

Comment:
There is a gap between the lines. See the ../docs/guides/pwd-optional-sign-in-email/nodeexpress/main/ page in the browser.

Reply:
fixed